databases/week1/lesson1.sql

```diff
@@ -0,0 +1,31 @@
+--1. Find out how many tasks are in the task table
+SELECT COUNT(*) FROM task
+
+--2.Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) FROM task WHERE due_date IS NULL
+
+--3.Find all the tasks that are marked as done
+SELECT task.id, task.title, status.name, status.id FROM task, status where status.name LIKE "Done" 
+AND task.status_id=status.id;
+
```

Comment:
A small fix and see if it gives different look than you query gives out. This may applies to next task too

```
SELECT task.id, task.title, status.name, status.id StatusId FROM task, status where status.name LIKE "Done" 
AND task.status_id=status.id;
```